tests/test_routes.py

```diff
@@ -462,11 +469,12 @@ def test_get_inventory_not_found(self):
         self.assertIn("was not found", data["message"])
 
         response = self.client.get(f"{BASE_URL}/0/FINAL")
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
```

Comment:
I had to change this because of how Flask-RestX works. It was throwing a 404 NOT FOUND if you pass in a bad condition enum